app/views/admin/categories/show.html.erb

```diff
@@ -1,7 +1,6 @@
 <div class="container main mt-5">
```

Comment:
We discussed that we do not need SHOW page for Admin Word. Only Index page.
There is error here if you force your WORDS CREATE function in controller to redirect_to ADMIN WORD SHOW PAGE
![image](https://user-images.githubusercontent.com/35232348/104156726-ad6ba180-5424-11eb-9511-914f12a40605.png)

So put all of this code from ADMIN WORD SHOW PAGE to ADMIN WORD INDEX.